sharding-core/sharding-core-route/src/main/java/org/apache/shardingsphere/core/route/spi/TimeService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.core.route.spi;
+
+import com.google.common.base.Optional;
+
+import java.util.Date;
+
+public interface TimeService {
+
+    /**
+     * get date from time server.
+     * @return date
+     */
+    Optional<Date> getTime();
```

Comment:
Why getTime return Optional? Which branch will return Optional date?

Reply:
I have changed it to `Date`